### DISCUSSION TITLE
Emoji warning - unsure what I'm meant to change

### DISCUSSION THREAD

**Discussion by StarfallProjects** [squidfunk/mkdocs-material#6207] [Q&A] [answered]
I just got the following warning:

> Material emoji logic has been officially moved into mkdocs-material
           version 9.4. Please use Material's 'material.extensions.emoji.to_svg'
           as mkdocs_material_extensions is deprecated and will no longer be
           supported moving forward. This is the last release.

These are the docs I follow for emoji support: https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/#configuration Nothing seems to have changes here compared to my current config.

What (if anything) do I need to do here?

**Comment by facelessuser** [comment, marked as answer]
Are you using `materialx.emoji.to_svg` or are you using `material.extensions.emoji.to_svg`?

**Reply by facelessuser** [reply]
I probably need to ensure the message references the original function name, but the deprecation warning states that you need to use `material.extensions.emoji.to_svg` moving forward.

I probably need to adjust the message when logged in MkDocs to be more clear by stating the original function name as well.

**Reply by StarfallProjects** [reply]
Oh doh I'm failing reading comprehension. I didn't see the detail change in the config lines in the docs.

**Reply by mrmanc** [reply]
We fell for the same, and it’s really not obvious what has changed, so I think a more explicit message would be fantastic 😃

**Reply by facelessuser** [reply]
Change is coming. Warnings will be far more explicit as to what needs to happen moving forward: https://github.com/facelessuser/mkdocs-material-extensions/pull/29

**Reply by facelessuser** [reply]
Deployed: https://pypi.org/project/mkdocs-material-extensions/. Thanks for the feedback.
